src/tabs/src/Tab.js

```diff
@@ -73,12 +106,14 @@ const Tab = memo(
 
     return (
       <Text
-        className={tabClassName}
+        className={themedClassName}
```

Comment:
Can we do `cx(className, themedClassName)` to make sure we support external classnames being passed in?